trusted-committer-article.md

```diff
@@ -0,0 +1,232 @@
+# InnerSource Learning Path: Trusted Committer
+
+## Introduction
+
+- tbd
+
+## Introducing The Trusted Committer Role
```

Comment:
I think it could be good _somewhere_ in the presentation to justify the need of why a trusted committer is necessary.  e.g. without the role you end up with an organically-grown pile of spaghetti code or people that would like to contribute but aren't able to actually finish a PR, etc.